src/classes/CON_Email.cls

```diff
@@ -178,8 +178,19 @@ public class CON_Email {
             this.valuedFields = new List<CON_EmailField>();
             Map<String, Schema.DescribeFieldResult> fieldMap = UTIL_Describe.getFieldsOfType('Contact', 'EMAIL');
             this.emailLabel = fieldMap.get('Email').getLabel();
+
+            Map<String, String> hedaEmailField = new Map<String, String>();
+            //build map for HEDA email field map (Label, API Name)
+            for(String fieldName : fieldMap.keySet() ) {
+                if(String.isNotBlank(UTIL_Namespace.getNamespace()) && fieldName.startsWithIgnoreCase(UTIL_Namespace.getNamespace())) {
```

Comment:
good point. I can verify this.

Reply:
Hi @ceiroa , I tested in the unmanaged org. The code still works. 

Actually, the issue we are trying to solve is that the org has multiple email fields with the same label. Such as HEDA 'Work Email' and NPSP 'Work Email'. 
In the unmanaged environment, system will not get confuse since usually the org will not have two email fields with the same label. Even it does happen somehow, user can always update the field label to avoid confusion since it is unmanaged package.